zerver/webhooks/travis/view.py

```diff
@@ -21,19 +21,26 @@
     u'Details: [changes]({}), [build log]({})'
 )
 
+def _should_pull_requests_be_ignored(ignore_param):
+    # type: (str) -> bool
+    return ignore_param.lower() == 'true'
+
 @api_key_only_webhook_view('Travis')
 @has_request_variables
 def api_travis_webhook(request, user_profile, client,
                        stream=REQ(default='travis'),
                        topic=REQ(default=None),
+                       ignore_pull_requests=REQ(default='true'),
```

Comment:
you can use `REQ(validator=check_bool, default=True)` here and have it come out as an actual Python bool.